pkg/sentry/config/config.go

```diff
@@ -0,0 +1,157 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	scheme "github.com/dapr/dapr/pkg/client/clientset/versioned"
+	dapr_config "github.com/dapr/dapr/pkg/config"
+	"github.com/dapr/dapr/utils"
+	log "github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	RootCertFilename            = "ca.crt"
+	IssuerCertFilename          = "issuer.crt"
+	IssuerKeyFilename           = "issuer.key"
+	kubernetesServiceHostEnvVar = "KUBERNETES_SERVICE_HOST"
+	kubernetesConfig            = "kubernetes"
+	selfHostedConfig            = "selfhosted"
+	defaultPort                 = 50001
+	defaultWorkloadCertTTL      = time.Hour * 24
+	defaultAllowedClockSkew     = time.Second * 20
```

Comment:
20 seconds is too short for clock skew. 20 seconds can be skewed between machines. I would set at least 15 mins.

Reply:
Good point.